.gitignore

```diff
@@ -1,3 +1,7 @@
 venom*.log
 test_results
-*.out
\ No newline at end of file
+*.out
+
+cmd/manager/apid.yml
+cmd/manager/config_create_test
+cmd/manager/config_test
```

Comment:
In case any of the tests fail, these directories might still exist. Just putting this in as a catch.